probe-rs/src/probe/ftdi/mod.rs

```diff
@@ -354,9 +353,9 @@ impl JtagAdapter {
         self.shift_ir(&ir.to_le_bytes(), irbits)?;
 
         let drbits = params.drpre + len_bits + params.drpost;
-        let request = if let Some(data) = data {
-            let mut data = BitVec::<Lsb0, u8>::from_slice(data);
-            data.truncate(len_bits);
+        let request = if let Some(data_slice) = data {
+            let data = BitSlice::<Lsb0, u8>::from_slice(data_slice).unwrap();
+            let mut data = BitVec::<Lsb0, u8>::from_bitslice(&data);
```

Comment:
Is the `truncate` call not needed anymore?

Reply:
I actually messed that up cause I tried a few different ways to make this work including creating a sub slice manually and somehow lost that :/ 
...fixed